# Assignment : 2 Basic Static level2, Set 2

#### Email:babasahebmane5050@gmail.com

In [4]:
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
import scipy.stats as stats
import seaborn as sns
import statsmodels.api as smf
import warnings
warnings.filterwarnings('ignore')

### Q1) The time required for servicing transmissions is normally distributed with µ = 45 minutes and σ = 8 minutes. The service manager plans to have work begin on the transmission of a customer’s car 10 minutes after the car is dropped off and the customer is told that the car will be ready within 1 hour from drop-off. What is the probability that the service manager cannot meet his commitment? 



Ans:
The service manager plans to work begin on the transmisssion of car 10 minutes after car is dropped off. so now µ becomes 55. 

mean = µ = 45+10 = 55.
standard deviation(std) = σ = 8.

In [7]:
df = stats.norm.sf(60, loc=55, scale=8)
print(" The probability that the service manager cannot meet his commitment", np.round(df,5))

 The probability that the service manager cannot meet his commitment 0.26599


### Q2) The current age (in years) of 400 clerical employees at an insurance claims processing center is normally distributed with mean µ = 38 and Standard deviation σ = 6. For each statement below, please specify True/False. If false, briefly explain why.

#### A) More employees at the processing center are older than 44 than between 38 and 44.

Ans:
mean = µ = 38

standard deviation = σ = 6

In [57]:
#p(x>40);  It means Probability of Employess older than 44 yeras of age.
prob_44=1-stats.norm.cdf(44, loc=38, scale=6)
prob_44

0.15865525393145707

In [58]:
print("Probability of Employess older than 44 yeras of age", np.round(prob_44*100,2),'%')

Probability of Employess older than 44 yeras of age 15.87 %


In [60]:
# P(38<X<44); It means Probability of Employees between 38 to 44 years of age.
prob_38_between_44=stats.norm.cdf(44,38,6)-stats.norm.cdf(38,38,6)
prob_38_between_44

0.3413447460685429

In [61]:
print("Probability of Employees between 38 to 44 years of age.", np.round(prob_38_between_44*100,2),'%')

Probability of Employees between 38 to 44 years of age. 34.13 %


Ans:

Answer is false, Because Probability of Employees between 38 to 44 years of age is 34.13 % which is more than Probability of Employess older than 44 yeras of age 15.86 %.

#### B) A training program for employees under the age of 30 at the center would be expected to attract about 36 employees.

In [17]:
stats.norm.cdf(30, loc=38, scale=6)

0.09121121972586788

In [18]:
print("A training program for employees under the age of 30 at the center would be expected to attract about", np.round(0.09121*400,2), '%')

A training program for employees under the age of 30 at the center would be expected to attract about 36.48 %


### Q3) If X1 ~ N(μ, σ2) and X2 ~ N(μ, σ2) are iid normal random variables, then what is the difference between 2 X1 and X1 + X2? Discuss both their distributions and parameters.       



Ans- As we know that if  X ~ N(µ1, σ1^2 ), and  Y ~ N(µ2, σ2^2 ) are two independent  random variables,

then  
X + Y = N(µ1 + µ2, σ1^2 + σ2^2 ) , and      X − Y = N(µ1 − µ2, σ1^2 + σ2^2 ) .

Similarly if  Z = aX + bY , where X and Y are as defined above, i.e Z is linear combination of X and Y , 

then, putting X and Y values in above Z equation,

Z = N(aµ1 + bµ2, a^2σ1^2 + b^2σ2^2 ).

Therefore in the question
2X1= N(2 u,4 σ^2) and 

X1+X2 =  N(µ + µ, σ^2 + σ^2 )

= N(2 u, 2σ^2 )
   
So Now difference between 2X1 and X1+X2

2X1-(X1+X2) = N( 4µ,6 σ^2).

### Q4) Let X ~ N(100, 20^2). Find two values, a and b, symmetric about the mean, such that the probability of the random variable taking a value between them is 0.99. 



Gievn:

mean = 100
    
standard deviation = std = 20

From the above details,we have to exclude .005% area from each
left and right tails. Hence, we want to find the 0.005th and the 
0.995th percentiles Z score values.

In [25]:
# find out z score for 0.005 percentiles
z_005 = np.round(stats.norm.ppf(0.005),2)
z_005

-2.58

In [27]:
# find out z score for 0.99 percentiles
z_099 = np.round(stats.norm.ppf(0.995),2)
z_099

2.58

In [38]:
# Now find out a and b values.
a = np.round((z_005*20)+100,1)
b = np.round((z_099*20)+100,1)
print("The Values of a and b :", a ,b)


The Values of a and b : 48.4 151.6


### Q5) Consider a company that has two different divisions. The annual profits from the two divisions are independent and have distributions Profit1 ~ N(5, 3^2) and Profit2 ~ N(7, 4^2) respectively. Both the profits are in  Doller Million. Answer the following questions about the total profit of the company in Rupees. 
Assume that $1 = Rs. 45

In [40]:
# Mean profits from two different divisions of a comapny = mean1+mean2
mean = 5+7
print("Mean Profit is Rs", mean*45, 'Million')

Mean Profit is Rs 540 Million


In [42]:
# Variance of profits from two different divisions f company = std^2 =std^1+std^2
std = np.sqrt((9)+(16))
print('Standard Deviation is Rs',std*45, 'Million')

Standard Deviation is Rs 225.0 Million


#### A) Specify a Rupee range (centered on the mean) such that it contains 95% probability for the annual profit of the company.

In [43]:
print('Range is Rs', (stats.norm.interval(0.95,540,225)), 'in Millions')

Range is Rs (99.00810347848784, 980.9918965215122) in Millions


#### B) Specify the 5th percentile of profit (in Rupees) for the company

In [46]:
#To compute 5th Percentile, we use the formula X=μ + Zσ; wherein from z table, 5 percentile = -1.645
x =540+(-1.645)*(225)
print("5th percentile of profit (in Rupees) is",  np.round(x,))

5th percentile of profit (in Rupees) is 170.0


#### C) Which of the two divisions has a larger probability of making a loss in a given year?

In [53]:
#Probability of division 1 making a loss
stats.norm.cdf(0,5,3)*100, "%"

(4.7790352272814705, '%')

In [50]:
#Probability of division making a loss
stats.norm.cdf(0,7,4)*100, '%'

(4.005915686381709, '%')

Ans:
    Division 1 has a larger probability of making loss which is 4.77%